spec/models/mechanic_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Mechanic, type: :model do
+  describe "relationships" do
+    it { should have_many(:ride_mechanics) }
```

Comment:
Nice job setting up your associations. You will also need to test the methods that you build in your models - `working_ride_list` is not tested.